java/pom.xml

```diff
@@ -692,9 +677,13 @@
         <module>vector</module>
         <module>tools</module>
         <module>adapter/jdbc</module>
-        <module>adapter/orc</module>
         <module>plasma</module>
         <module>flight</module>
+        <module>performance</module>
```

Comment:
Should we add `algorithm` too?
`/project/modules/` includes `algorithm`.

Reply:
fixed now. Turns out that maven appends the ones mentioned in the profile - so, I just added the two required (gandiva and adapter/orc)